main-app/app/index.html

```diff
@@ -5,15 +5,18 @@
     <title>Noughts and Crosses</title>
     <link rel="stylesheet" href="css/main.min.css" type="text/css">
 </head>
-<body>
+<body ng-class="{'theme-1': currentTheme===1, 'theme-2': currentTheme===2, 'theme-3': currentTheme===3}"  ng-controller="MainController">
```

Comment:
you could do this with class = '{{currentTheme | *themeFilter*}}'
Where *themeFilter* is a filter that returns the class name of the style you want.
Similar JS fiddle: http://jsfiddle.net/ja3h8/2/
This has the advantage of making pushing the conversion from property to className into something unit-testable. Another option would be to have the model expose the correct class name for the theme - I encapsulated this into a themeProvider, which handles switching etc.